Configuration option "--disable-imageProcessor-build" disables use of
proprietary libImageProcessor which is delivered pre-compiled so unusable for
illumos.  Skip linking with ImageProcessor library.

Not for upstream.

--- hplip-3.25.6/Makefile.am.orig
+++ hplip-3.25.6/Makefile.am
@@ -598,7 +598,7 @@
 	prnt/hpcups/ImageProcessor.h
 
 hpcups_CXXFLAGS = $(APDK_ENDIAN_FLAG) $(DBUS_CFLAGS)
-hpcups_LDADD = -L./prnt/hpcups/ -ljpeg -ldl -lImageProcessor -lcups -lcupsimage -lz $(DBUS_LIBS)
+hpcups_LDADD = -L./prnt/hpcups/ -ljpeg -ldl -lcups -lcupsimage -lz $(DBUS_LIBS)
 #else
 #hpcupsdir = $(cupsfilterdir)
 #hpcups_PROGRAMS = hpcups
